### PR TITLE
add required field in the minio spec for specifying secret

### PR DESCRIPTION
**Pull request by soulseen**
add required field in the minio spec for specifying secret

Signed-off-by: soulseen <zhuxiaoyang1996@gmail.com>

### CODE REVIEW

File: pkg/cluster/controllers/storage/minio.go [hunk 1]
```diff
@@ -224,7 +224,7 @@ func (m *MinIOController) getMinIONamespacedName() types.NamespacedName {
 func (m *MinIOController) getMinIOSecretNamespacedName() types.NamespacedName {
 	return types.NamespacedName{
 		Namespace: m.HarborCluster.Namespace,
-		Name:      DefaultPrefix + m.HarborCluster.Name + "-" + DefaultCredsSecret,
+		Name:      m.HarborCluster.Spec.InClusterStorage.MinIOSpec.SecretRef,
```
Comment (steven-zou) [inline pkg/cluster/controllers/storage/minio.go:227]: @soulseen 

Can we make this field be an optional item? When the user does not provide, we keep the original default behavior that is auto-generating a random secret.

**Review by steven-zou — COMMENTED** [review]
@soulseen what do u think my advice?

File: pkg/cluster/controllers/storage/minio.go (hunk 1 above)
Comment (soulseen) [reply pkg/cluster/controllers/storage/minio.go]: agree, and updated.

**Review by steven-zou — APPROVED** [review]
LGTM

**Review by bitsf — APPROVED** [review]

**Pull request merged**